src/plugins/telemetryTable/TelemetryTableViewProvider.js

```diff
@@ -54,15 +54,13 @@ define([
             view(domainObject, objectPath) {
                 let table = new TelemetryTable(domainObject, openmct);
                 let component;
-
                 let markingProp = {
                     enable: true,
                     useAlternateControlBar: false,
                     rowName: '',
                     rowNamePlural: ''
                 };
-
-                return {
+                const viewProvider = {
```

Comment:
This is a __view__, not a view provider. The thing returning it is a view provider.